+ source ../common/common_lib.sh
++ source /root/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Tue Dec  6 02:34:21 2022 - INFO  - Start environmental preparation.
+ DNF_INSTALL udisks2
+ pkgs=udisks2
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs udisks2 --node 1 --tempfile ''
+ tmpfile2=/tmp/tmprbaitbkq
+ '[' -z '' ']'
+ tmpfile=/tmp/tmprbaitbkq
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Tue Dec  6 02:35:54 2022 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Tue Dec  6 02:35:55 2022 - INFO  - Start testing...
+ test_execution udisks2.service
+ service=udisks2.service
++ date '+%Y-%m-%d %T'
+ log_time='2022-12-06 02:35:56'
+ test_restart udisks2.service
+ service=udisks2.service
+ systemctl restart udisks2.service
A dependency job for udisks2.service failed. See 'journalctl -xe' for details.
+ CHECK_RESULT 1 0 0 'udisks2.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='udisks2.service restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'udisks2.service restart failed'
+ LOG_ERROR 'udisks2.service restart failed'
+ message='udisks2.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'udisks2.service restart failed'
Tue Dec  6 02:35:57 2022 - ERROR - udisks2.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Tue Dec  6 02:35:58 2022 - ERROR - ../common/common_lib.sh line 34
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status udisks2.service
+ grep 'Active: active'
+ CHECK_RESULT 1 0 0 'udisks2.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='udisks2.service restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'udisks2.service restart failed'
+ LOG_ERROR 'udisks2.service restart failed'
+ message='udisks2.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'udisks2.service restart failed'
Tue Dec  6 02:36:05 2022 - ERROR - udisks2.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Tue Dec  6 02:36:06 2022 - ERROR - ../common/common_lib.sh line 37
+ return 0
+ systemctl stop udisks2.service
+ CHECK_RESULT 0 0 0 'udisks2.service stop failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='udisks2.service stop failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status udisks2.service
+ grep 'Active: inactive'
     Active: inactive (dead)
+ CHECK_RESULT 0 0 0 'udisks2.service stop failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='udisks2.service stop failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl start udisks2.service
A dependency job for udisks2.service failed. See 'journalctl -xe' for details.
+ CHECK_RESULT 1 0 0 'udisks2.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='udisks2.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'udisks2.service start failed'
+ LOG_ERROR 'udisks2.service start failed'
+ message='udisks2.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'udisks2.service start failed'
Tue Dec  6 02:36:13 2022 - ERROR - udisks2.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Tue Dec  6 02:36:14 2022 - ERROR - ../common/common_lib.sh line 44
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status udisks2.service
+ grep 'Active: active'
+ CHECK_RESULT 1 0 0 'udisks2.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='udisks2.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'udisks2.service start failed'
+ LOG_ERROR 'udisks2.service start failed'
+ message='udisks2.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'udisks2.service start failed'
Tue Dec  6 02:36:21 2022 - ERROR - udisks2.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Tue Dec  6 02:36:22 2022 - ERROR - ../common/common_lib.sh line 47
+ return 0
+ test_enabled udisks2.service
+ service=udisks2.service
++ systemctl is-enabled udisks2.service
+ state=enabled
+ '[' enabled == enabled ']'
++ systemctl disable udisks2.service
++ grep Removed
++ awk '{print $2}'
++ awk '{print substr($0,1,length($0)-1)}'
+ symlink_file=/etc/systemd/system/graphical.target.wants/udisks2.service
+ find /etc/systemd/system/graphical.target.wants/udisks2.service
find: ‘/etc/systemd/system/graphical.target.wants/udisks2.service’: No such file or directory
+ CHECK_RESULT 1 0 1 'udisks2.service disable failed'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='udisks2.service disable failed'
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ systemctl enable udisks2.service
Created symlink /etc/systemd/system/graphical.target.wants/udisks2.service → /usr/lib/systemd/system/udisks2.service.
+ find /etc/systemd/system/graphical.target.wants/udisks2.service
/etc/systemd/system/graphical.target.wants/udisks2.service
+ CHECK_RESULT 0 0 0 'udisks2.service enable failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='udisks2.service enable failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ journalctl --since '2022-12-06 02:35:56' -u udisks2.service
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
Dec 06 02:35:56 anolis-riscv systemd[1]: Dependency failed for udisks2.service - Disk Manager.
Dec 06 02:35:56 anolis-riscv systemd[1]: udisks2.service: Job udisks2.service/start failed with result 'dependency'.
Dec 06 02:36:12 anolis-riscv systemd[1]: Dependency failed for udisks2.service - Disk Manager.
Dec 06 02:36:12 anolis-riscv systemd[1]: udisks2.service: Job udisks2.service/start failed with result 'dependency'.
+ CHECK_RESULT 0 0 1 'There is an error message for the log of udisks2.service'
+ actual_result=0
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of udisks2.service'
+ '[' -z 0 ']'
+ '[' 1 -eq 0 ']'
+ test 0x == 0x
+ test -n 'There is an error message for the log of udisks2.service'
+ LOG_ERROR 'There is an error message for the log of udisks2.service'
+ message='There is an error message for the log of udisks2.service'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'There is an error message for the log of udisks2.service'
Tue Dec  6 02:36:30 2022 - ERROR - There is an error message for the log of udisks2.service
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 28'
+ message='../common/common_lib.sh line 28'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 28'
Tue Dec  6 02:36:30 2022 - ERROR - ../common/common_lib.sh line 28
+ return 0
+ test_reload udisks2.service
+ service=udisks2.service
+ systemctl start udisks2.service
A dependency job for udisks2.service failed. See 'journalctl -xe' for details.
+ systemctl reload udisks2.service
+ grep 'Job type reload is not applicable'
Failed to reload udisks2.service: Job type reload is not applicable for unit udisks2.service.
+ CHECK_RESULT 0 0 0 'Job type reload is not applicable for unit udisks2.service'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Job type reload is not applicable for unit udisks2.service'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl status udisks2.service
+ grep 'Active: active'
+ systemctl status udisks2.service
+ grep 'inactive (dead)'
     Active: inactive (dead)
+ systemctl status udisks2.service
+ grep skip
+ grep 'Condition check'
+ CHECK_RESULT 1 0 0 'udisks2.service reload causes the service status to change'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='udisks2.service reload causes the service status to change'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'udisks2.service reload causes the service status to change'
+ LOG_ERROR 'udisks2.service reload causes the service status to change'
+ message='udisks2.service reload causes the service status to change'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'udisks2.service reload causes the service status to change'
Tue Dec  6 02:36:33 2022 - ERROR - udisks2.service reload causes the service status to change
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 95'
+ message='../common/common_lib.sh line 95'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 95'
Tue Dec  6 02:36:34 2022 - ERROR - ../common/common_lib.sh line 95
+ return 0
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Tue Dec  6 02:36:35 2022 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 6
+ test 6 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Tue Dec  6 02:36:36 2022 - ERROR - failed to execute the case.
+ exit 6
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Tue Dec  6 02:36:37 2022 - INFO  - start environment cleanup.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmprbaitbkq ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmprbaitbkq
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Tue Dec  6 02:36:57 2022 - INFO  - Finish environment cleanup!
